src/test/scala/viper/gobra/GobraPackageTests.scala

```diff
@@ -99,7 +100,10 @@ class GobraPackageTests extends GobraTests {
       // set throwError to true: Scallop will throw an exception instead of terminating the program in case an
       // exception occurs (e.g. a validation failure)
       throwError.value = true
-      Some(new ScallopGobraConfig(args.toSeq).config)
+
+      // Simulate pick of package, Gobra normally does
+      val config = new ScallopGobraConfig(args.toSeq).config
+      Some(config.copy(inputs = config.inputPackageMap.values.head))
```

Comment:
why do you need to modify the inputs?

Reply:
Since the feature to pass a package as input doesn't exist anymore, it is handled via an input directory combined with the -p option. Here we have to mimic the behaviour of the `verifyAllPackages` method to make this test work.